client/test/accounts_lib.spec.js

```diff
@@ -0,0 +1,95 @@
+'use strict'
+
+const assert = require('chai').assert
+
+const AccountsLib = require('../app/libs/accounts')
+
+
+describe("Accounts libs spec", () => {
+
+    describe("Validate and returns a new state", () => {
+
+        it("should returns a simple value in a new state", () => {
+            let state = {}
+            let nextState = AccountsLib.validateState('login', 'mail@cozy.io', state)
```

Comment:
Coz' in some tests I need to change their values (e.g. https://github.com/cozy/cozy-emails/pull/866/files/769ab196b3f04156aa8bbc009ccf4ce34e6f3779#diff-75f7b93415035b03c811d0173ae2f7fbR58) so I try to stay consistent and use `let` everywhere. But I'm open to another solution if you provide one :smile: